x-pack/plugins/security_solution/public/app/home/template_wrapper/index.tsx

```diff
@@ -73,11 +77,16 @@ export const SecuritySolutionTemplateWrapper: React.FC<SecuritySolutionPageWrapp
     const { show: isShowingTimelineOverlay } = useDeepEqualSelector((state) =>
       getTimelineShowStatus(state, TimelineId.active)
     );
+    const endpointMetadataIndex = useMemo<string[]>(() => {
+      return [ENDPOINT_METADATA_INDEX];
+    }, []);
+    const [, { indexExists: metadataIndexExists }] = useFetchIndex(endpointMetadataIndex, true);
+    const { indicesExist } = useSourcererScope();
+    const securityIndicesExist = indicesExist || metadataIndexExists;
 
-    /* StyledKibanaPageTemplate is a styled EuiPageTemplate. Security solution currently passes the header and page content as the children of StyledKibanaPageTemplate, as opposed to using the pageHeader prop, which may account for any style discrepancies, such as the bottom border not extending the full width of the page, between EuiPageTemplate and the security solution pages.
-     */
+    // StyledKibanaPageTemplate is a styled EuiPageTemplate. Security solution currently passes the header and page content as the children of StyledKibanaPageTemplate, as opposed to using the pageHeader prop, which may account for any style discrepancies, such as the bottom border not extending the full width of the page, between EuiPageTemplate and the security solution pages.
 
-    return (
+    return securityIndicesExist ? (
```

Comment:
@stephmilovic I'm happy to remove the checks downstream, however note that this check does include one more index with the metadata index, so show/hiding this Add Data screen has one additional check.  Thoughts?  Should we still remove all downstream checks?

> Also, what about the rules page? currently, we will show it even if no index exists

We also have a couple pages that displayed without having data such as Trusted Apps and Event Filters.  The intention here is to have a consistent "Empty State" across all apps.  Based on [this conversation](https://github.com/elastic/kibana/pull/108763#discussion_r709315413) with @cchaos I think the UX/Design team is open to more granular "Empty State" views in apps in the next iteration.

Reply:
Yep, the next step will the the in-page empty states. We're working our way through the user's journey. 😄

Reply:
ok, so if your check is `indicesExist || metaIndicesExist` and we will want to display the `<OverviewEmpty />` page if `indicesExist = false` and `metaIndicesExist = true` vs your new Empty State page? That's not consistent right? Why not make it an `&&` condition and we can get rid of the downstream checks and `<OverviewEmpty />`?

Reply:
@stephmilovic yes, I see your point on things not being consistent.  The condition should still be `indicesExist || metaIndicesExist` because our Endpoint management pages should be usable even if we only have Endpoint metadata.  To keep things consistent, I did go ahead and removed the downstream checks.

You can see it in this commit : https://github.com/elastic/kibana/pull/112142/commits/7de5044cdc30faea1acebe0f0ebac875ce64d356

let me know if this looks OK

Reply:
@stephmilovic and I spoke offline.

We'd like the same empty state to still be available for the other pages in the event that we have `metaIndicesExist  = true` and `indicesExist  = false` .  The new empty state replaces the old one, so that is how it's implemented with the downstream checks to `indicesExist` .   I reverted the changes above to put those checks back.

Commit: https://github.com/elastic/kibana/pull/112142/commits/27c39b721871f0313153aa9df0421b3dfc9b1f05